app/core/lifespan.py

```
@@ -63,6 +73,39 @@ async def create_lifespan(app: FastAPI) -> AsyncIterator[None]:
+            runner = ConnectorRunner(event_service_factory=factory)
+            if settings.connector_email_enabled and settings.connector_email_tenant_id:
+                runner.register(
+                    EmailConnector(
+                        connector_id="email",
+                        tenant_id=settings.connector_email_tenant_id,
+                        poll_interval_seconds=settings.connector_email_poll_interval_seconds,
+                    )
+                )
+            # CDC, Kafka, file_watch: register when implemented
+            app.state.connector_runner = runner
+            asyncio.create_task(runner.start_all(), name="connector_runner")
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)